src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Approve.tsx

```diff
@@ -74,7 +74,7 @@ export const Approve: React.FC<ApproveProps> = ({ accountId, onNext, isReset })
     assetReference,
   })
   const assets = useAppSelector(selectAssets)
-  const asset: Asset | undefined = useAppSelector(state => selectAssetById(state, assetId ?? ''))
+  const asset: Asset | undefined = useAppSelector(state => selectAssetById(state, assetId))
```

Comment:
might as well get this while here too
```suggestion
  const asset = useAppSelector(state => selectAssetById(state, assetId))
```

Reply:
bfd29693179ec4c7452be59617c5e2fd6f75684c